iphone/Classes/TiUIListView.m

```diff
@@ -544,6 +546,13 @@ -(NSInteger)sectionForSearchSection:(NSInteger)section
 -(void)setSeparatorInsets_:(id)arg
 {
     [self tableView];
+    DEPRECATED_REPLACED(@"UI.ListView.separatorInsets", @"5.2.0", @"UI.ListView.tableSeparatorInsets");
+    [self setTableSeparatorInsets_:arg];
+}
+-(void)setTableSeparatorInsets_:(id)arg
+{
+    [self tableView];
```

Comment:
We should not call it here, since it just creates the instance of not available already. Change `_tableView` to `[self tableView]` below and in the other cases you used it.